shenyu-admin/src/main/java/org/apache/shenyu/admin/service/impl/UpstreamCheckService.java

```diff
@@ -144,6 +144,9 @@ public void setup() {
             this.fetchUpstreamData();
             executor = new ScheduledThreadPoolExecutor(1, ShenyuThreadFactory.create("scheduled-upstream-task", false));
             scheduledFuture = executor.scheduleWithFixedDelay(this::scheduled, 10, scheduledTime, TimeUnit.SECONDS);
+        } else {
```

Comment:
check is false why need new ScheduledThreadPoolExecutor ?

Reply:
hi  thanks for you， i will fix it， because have other error

Reply:
welcome